src/components/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+
+import React from "react";
+import './Button.css'
+
+interface ButtonProps {
+  submit: () => void;
+  buttonText?: string; 
+}
+
+
+
+export const Button = ({submit, buttonText}: ButtonProps) =>{
+return (
+  // modal__form-submit
+  <div className="modal__form-submit">
+          <button
+          // modal__form-submit__button
+            className="relative w-60 h-16 rounded-2xl border-4 border-[#cf554e] bg-[#cf554e]"
```

Comment:
There is also a button that doesn't follow this type of styling. 
Look at the Figma and find "Learn more about us", "Staffing services" or "Renting Equipment" need to account for these buttons too.

In addition, for this specific style of button, there is a hover state which wasn't coded in. 

Lastly, you need to use the names for the colors from the tailwind config. You shouldn't use hex.

Reply:
Are you referring to the curor not being a pointer @RPinkha ? Because I did notice that when hovering on the button the pointer seemed a little off. It only appeared on a specific small part of the button.

Reply:
> Are you referring to the cursor not being a pointer @RPinkha ? Because I did notice that when hovering on the button the pointer seemed a little off. It only appeared on a specific small part of the button.

No. I am referring to the hover styles in their entirety. 
There is a hover style for the specific button he coded in which he needs to apply, and there is also no pointer: cursor change like you mentioned.